README.md

```diff
@@ -21,35 +21,60 @@ love to have you! :balloon:
 
 Here's how to join:
 
-1. **Donate $2,000 per full-time developer on staff** to Open Source projects of
-   your choice, and commit to doing so in future years. The projects you're
-   donating to should meet the [Open Source Definition][osd]. Of course, this
-   includes any existing donations you've made this fiscal year. You can donate
-   to any projects you like, but if you need help figuring out which projects
-   you depend on, you can use a tool like [Thanks.dev](https://thanks.dev/).
-2. **Create a short JSON file with your company and donation info**, and host it
+1. **Donate $2,000 per full-time equivalent developer on staff** to Open Source
+   maintainers or foundations of your choice, and commit to doing so in future
+   years. The projects you're donating to should meet the [Open Source
+   Definition][osd]. Of course, this includes any existing donations you've made
+   this fiscal year. You can donate to any projects you like, but if you need
```

Comment:
Doesn't need to be fiscal year after all, I think.

Reply:
Since we had fiscal already, not a blocker for this PR. We can follow up.

Reply:
Maybe we can make it clear that it can be any arbitrary year? Because it's not necessarily a calendar year either — Sentry's `dateYearEnding` is e.g. `2024-01-31`.

Reply:
Should be good in #58.